servers/sanat/roles/hhvm/tasks/main.yml

```diff
@@ -10,6 +10,9 @@
     repo_gpgcheck: no
     enabled: yes
 
+- name: Enable Epel repository
```

Comment:
I started a fresh Centos virtual machine, and it was unable to install HHVM without epel